### ISSUE TITLE
High Contrast: Hovering over list items and header should highlight the list item background

### ISSUE THREAD

**Issue by bcoard** [microsoft/fluentui#13491]
### Environment Information

- **Browser and OS versions**: Edge (Chromium) & Window 10 2004

### Describe the issue:

In high contrast modes, hovering or focusing on a list item and it elements should highlight its background as well as give them a focus/hover border. Selecting list items should do the same, without the focus/hover border. That latter appears as though it's already being done.

### Please provide a reproduction of the issue in a codepen:

1. Go To https://developer.microsoft.com/en-us/fluentui#/controls/web/groupedlist.
2. Turn on high contrast black in Windows.
3. Hover over or focus on list items and headers.

NOTE: List items in the demo page are being highlighted on focus because the list item is automatically being selected. If selection by default isn't enabled the highlight will not occur.

#### Actual behavior:

List items and headers do not highlight their backgrounds and give a focus/hover border when focused or hovered.

#### Expected behavior:

List items and headers should be highlighted and have a focus/hover border when focused and hovered.

**Comment by paulgildea** [comment]
@xugao is this related to the overall high contrast in Edge bug? #11742?

**Comment by xugao** [comment]
@paulgildea - this is a high contrast bug and it repros on both old/new Edge

**Comment by bcoard** [comment]
@paulgildea Let me know if you want to discuss this with me a bit more in depth. I filed this bug to help better align with Office designs and feedback.
